server.js

```diff
@@ -124,7 +127,17 @@ app.post("/updateUserTag",async function(req, res){
     if (accessToken != '' && userAddress != '')
         await db.updateUserTag(userAddress, accessToken, tag)
     res.send({'status': 'success'});
-})
+});
+
+app.post('/formData', upload.single('imageURL'), async function (req, res) {
+    var name = req.body.organizationName
+    var address = req.body.address
+    var imageURL = 'http://127.0.0.1:3000/' + req.file.path;
```

Comment:
Don't hard code the address , see if you can use fs library like they did in this blog. If nothing works , create a global variable for this.

https://medium.com/@otoloye/uploading-files-to-aws-s3-using-nodejs-multer-mongodb-and-postman-part-1-de790b8131d4